org.mixedrealitytoolkit.input/Assets/Prefabs/MRTK XR Rig.prefab

```diff
@@ -28,13 +28,13 @@ Transform:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 2351505566771328562}
+  serializedVersion: 2
```

Comment:
That was my original approach, however, I noticed that approach would have required me to update each scene with the new prefabs making this PR bigger so I decided for the smaller PR approach.
As for the changes of the Obsolete prefabs:
1 - I went with the Obsolete* renaming because I found a prefab that had such renaming (org.mixedrealitytoolkit.input\Assets\Prefabs\Obsolete MRTK Hand Controller.prefab) and wanted to be consistent with how it has been done before.
2 - The obsolete prefabs content was modified not by me but by the upgrade tool when upgrading to packages' versions, in particular, the com.unity.xr.interaction.toolkit 3.0.1 package modified the prefabs containing an XRController component or those in which the XRController functionality was transferred to.  For example, on the left we have the pre XRI 3 migration prefab and on the right we have the same prefab after com.unity.xr.interaction.toolkit package version was bumped to 3.0.1
<img width="491" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/c033e78c-b506-40a3-b7d3-c1d21bf5a87f">